README.md

```diff
@@ -236,6 +236,11 @@ while True:
         break
 ```
 
+### AgentScope Studio
```

Comment:
Maybe better to show an attractive picture, a code block, and a brief intro (no need to elaborate the usage)?

```bash
as_studio path/to/your/script.py
```